README.md

```diff
@@ -19,7 +19,9 @@ run on other Python 3 platforms as well.
 
 ## Release Status
 
-Alpha
+Beta
 
 ### Supported Python Versions
 Python >= 3.6
```

Comment:
Is this still true? I know work was underway to support more versions of python.

Reply:
It is still true, but work is underway: https://github.com/googleapis/python-ndb/issues/189